jekyll/_cci2/ssh-access-jobs.md

```diff
@@ -41,72 +41,57 @@ would run: `ssh -p <remote_ip> -- powershell.exe`. Consider reading the [Hello W
 
 The build VM will remain available for an SSH connection for **10 minutes after the build finishes running** and then automatically shut down (or you can cancel it). After you SSH into the build, the connection will remain open for **one hour** for customers on a free plan or **two hours** for all other customers.
 
-**Note**: If your job has parallel steps, CircleCI launches more than one VM to perform them. Thus, you'll see more than one 'Enable SSH' and 'Wait for SSH' section in the build output.
+**Note**: If your job has parallel steps, CircleCI launches more than one VM to perform them. Thus, you will see more than one 'Enable SSH' and 'Wait for SSH' section in the build output.
 
 ## Debugging: "permission denied (publickey)"
 {: #debugging-permission-denied-publickey }
 
-If you run into permission troubles trying to SSH to your job, try
-these things:
+If you run into permission troubles trying to SSH to your job, try these things:
 
 ### Ensure authentication with GitHub/Bitbucket
 {: #ensure-authentication-with-githubbitbucket }
-{:.no_toc}
 
-A single command can be used to test that your keys are set up as expected. For
-GitHub run:
+A single command can be used to test that your keys are set up as expected. 
 
+For GitHub run
 ```bash
 ssh git@github.com
 ```
 
 or for Bitbucket run:
-
 ```bash
 ssh -Tv git@bitbucket.org
 ```
 
 and you should see:
-
 ```bash
 $ Hi :username! You've successfully authenticated...
 ```
 
-for GitHub or for Bitbucket:
-
+as well as:
 ```bash
 $ logged in as :username.
 ```
 
-If you _don't_ see output like that, you need to start by
+If you _do not_ see output as described above, you need to start by:
 [troubleshooting your SSH keys with GitHub](https://help.github.com/articles/error-permission-denied-publickey)/
 [troubleshooting your SSH keys with Bitbucket](https://confluence.atlassian.com/bitbucket/troubleshoot-ssh-issues-271943403.html).
 
 ### Ensure authenticating as the correct user
 {: #ensure-authenticating-as-the-correct-user }
-{:.no_toc}
 
-If you have multiple accounts, double-check that you are
-authenticated as the right one!
+If you have multiple accounts, double-check that you are authenticated as the right one!
```

Comment:
Nitpick (and also we don't have this in our style guide): would we want to stick with the exclamation points? I don't typically see it used in docs... Also thinking about being consistent with the rest of the section (i.e. "If you are authenticating as the wrong user...")